### DISCUSSION TITLE
Adding images, tables, equatitions as annotations

### DISCUSSION THREAD

**Discussion by GDLMadushanka** [sumatrapdfreader/sumatrapdf#3287] [Ideas]
I'm currently working on something like this to help the readers to not scroll above and look for previous figures, equitation s and ect. 
![graphic1](https://cloud.githubusercontent.com/assets/8251118/7953431/3732dd7c-09e2-11e5-8147-a5771b1b59be.jpg)

**Comment by GitHubRulesOK** [comment]
@GDLMadushanka 
Did you find a way to provide the above, 

There are commercial products that do PDF or ePub pop-ups like your illustration, but sumatrapdf can not run their code.

**Comment by GitHubRulesOK** [comment]
Migrated from Issues to Ideas (Feature Proposal)
 as popup/popout modal previews are not a natural feature for static eBook Fixed Format viewers,
 although some may have such a non standard enhancement
